dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/event/RumEventDeserializer.kt

```diff
@@ -81,6 +82,7 @@ internal class RumEventDeserializer : Deserializer<RumEvent> {
             EVENT_TYPE_RESOURCE -> ResourceEvent.fromJson(jsonString)
             EVENT_TYPE_ACTION -> ActionEvent.fromJson(jsonString)
             EVENT_TYPE_ERROR -> ErrorEvent.fromJson(jsonString)
+            EVENT_TYPE_LONG_TASK -> LongTaskEvent.fromJson(jsonString)
```

Comment:
this may not be really needed, because `deserialize` is used only in NDK from and only for View events (from RUM events scope), but since other RUM event types are here, I added it for the symmetry with serializer.

Reply:
It's mostly there to ensure we don't break things if someone tries to use the deserializer